### PR TITLE
Update SuperPMI readme

### PR DESCRIPTION
**Pull request by BruceForstall**
Give many more details about the collection process.

### CODE REVIEW

**Comment by BruceForstall** [comment]
cc @dotnet/jit-contrib

File: src/coreclr/src/ToolBox/superpmi/readme.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+impact of a JIT code change.
+
+## SuperPMI architecture
+
+SuperPMI works in two phases: collection and playback.
+
+In the collection phase, the system is configured to collect SuperPMI data.
+Then, run any set of .NET managed programs. When these managed programs invoke the JIT
+compiler, SuperPMI gathers and captures all information passed between the
+JIT and its .NET host. This data is post-processed to remove essentially
```
Comment (sandreenko) [inline src/coreclr/src/ToolBox/superpmi/readme.md]: Do we call our VM/EE .NET host?

**Review by sandreenko — APPROVED** [review]
LGTM, thank you.

File: src/coreclr/src/ToolBox/superpmi/readme.md (hunk 1 above)
Comment (BruceForstall) [reply src/coreclr/src/ToolBox/superpmi/readme.md]: That's a bit of an odd phrasing that was in the original readme.txt, that I didn't change.

**Pull request merged**